lib/runtime-core/src/error.rs

```diff
@@ -208,6 +209,8 @@ impl std::fmt::Display for RuntimeError {
                     write!(f, "\"{}\"", s)
                 } else if let Some(s) = data.downcast_ref::<&str>() {
                     write!(f, "\"{}\"", s)
+                } else if let Some(exc_code) = data.downcast_ref::<ExceptionCode>() {
+                    write!(f, "\"{:?}\"", exc_code)
```

Comment:
Maybe we should write it is an exception code, so that the user has more insight than the variant name. My suggestion:

```rust
write!(f, "exception code \"{:?}\"", exc_code)
```

Reply:
Thanks! Fixed and now the error message looks like `Error: RuntimeError: Caught exception of type "Arithmetic".`.